tcpdf.php

```diff
@@ -1641,6 +1641,20 @@ class TCPDF {
 	 */
 	protected $svgdefs = array();
 
+    /**
```

Comment:
I am afraid this patch is not very minimal and so will have less changes being accepted into tcpdf

Reply:
This library is part of the moodle core. We at TU Graz are relying on it to generate certificates, which among other contents contain logos of our partners (SVGs). Its crucial for us that they are correct on the pdf and currently this is not the case. Hence our contribution to this library.
Is there anything that can be done to increase chances? I will change up the contribution in any way necessary to provide this functionality. (:

Reply:
First, thank you for contributing to the libraries you use !

Second, I am here as an advisor and try to monitor issues and pull-requests on my free time
Currently the library owner and maintainer remains mysterious in choices that are made. But it's mostly the rule of more the change is okay and small more it has chances to get on a release one day.

So from time to time I build an issue with a summary of PRs: https://github.com/tecnickcom/TCPDF/issues/634
Maybe it helps, I never know, nobody knows
So it's a wait and see, but you need to get it slim and clean and right. Or it will probably get refused and you are back in a 6-8 months loop to get feedback on the new changes

This recent PR changed the library maintenance official statement, and that's a relief: #589 

In summary, I think this patch will appear too large. Is there other ways to change less code ?